### PR TITLE
LSX v2 moment

### PR DESCRIPTION
**Pull request by xamionex**
So people have been getting this Authentication Error when they click Launch Northstar and LSX fix hasn't been working for them, this is the alternative way. Still seeing if LSX truly does not work for them.

### CODE REVIEW

**Comment by xamionex** [comment]
Ignore every commit except last, dunno why it lists all those

**Comment by GeckoEidechse** [comment]
I'm assuming the reason for the additional section in Troubleshooting is regarding Emma's investigation into an additional LSX bug?

> Ignore every commit except last, dunno why it lists all those

That's due to me squashing commits on merge but you still working on the same old branch. Therefore on merge git sees the "missing" commits and tries to add them.

File: docs/installing-northstar/troubleshooting.md [hunk 1]
```diff
@@ -142,3 +142,16 @@ Campaign is not supported in 1.4, but you could do one of these **"_fixes_"**
     - [If you can't open the console](#i-cant-open-the-console)
 2. Adding `-vanilla` to your launch options
     - [How to add launch options](#adding-launch-options)
+
+## Authentication Failed when clicking Launch Northstar
+
+Before trying this check out [this](#failed-to-initialize-origin-origin-core-seems-to-be-running-but-the-lsx-authentication-challenge-fai)\
+If that didn't work DM me on discord (Petar#0389) and tell me if it did or didn't work.\
```
Comment (GeckoEidechse) [inline docs/installing-northstar/troubleshooting.md]:
```suggestion
Before trying this check out [this section](#failed-to-initialize-origin-origin-core-seems-to-be-running-but-the-lsx-authentication-challenge-fai).\
If that didn't work DM `Petar#0389` on Discord and tell him if it did or did not work.\
```

**Comment by xamionex** [comment]
No It's a fix I came up with on the spot, and It seems to be working 😅
every time... it just _works_

**Comment by GeckoEidechse** [comment]
Hmm, aight. If you got enough people messaging you know that it does (not) work, LMK, so can remove the part about messaging you later again.

Telling someone who's troubleshooting an issue to DM a specific person on Discord isn't really fitting for something in a Wiki (it should just describe the solution) but I guess it's fine as a temporary addition to gauge the effectiveness of the fix ^^

**Pull request merged**